lib/overcommit/hook/pre_commit/php_stan.rb

```diff
@@ -0,0 +1,30 @@
+module Overcommit::Hook::PreCommit
+  # Runs `phpstan` against any modified PHP files.
+  # For running `phpstan` with Laravel, it requires setup with `ide_helper`.
+  #
+  # References:
+  # https://github.com/phpstan/phpstan/issues/239
+  # https://gist.github.com/edmondscommerce/89695c9cd2584fefdf540fb1c528d2c2
+  class PhpStan < Base
+    MESSAGE_REGEX = /^(?<file>.+)\:(?<line>\d+)\:(?<message>.+)/
+
+    def run
+      messages = []
+
+      applicable_files.each do |file|
+        result = execute(command, args: [file])
+        output = result.stdout.chomp
+        if result.status
```

Comment:
`status` is a number, which will always be truthy, which I don't think is what you intended here. Let's use

```ruby
result.success?
```

Reply:
Thanks. Will update.